.travis.yml

```diff
@@ -5,9 +5,6 @@ os:
   - linux
   # - osx
 julia:
```

Comment:
I think we should just put 
```
julia: 
    - 1
``` 
This way travis will use the latest release only (1.4 as of today).